test/jbuilder/schema/builder_test.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class JbuilderSchema::BuilderTest < ActiveSupport::TestCase
+  include JbuilderSchema
+
+  test "renders a schema from a fixture to yaml" do
+    user = User.new(id: 1, email: "someone@example.org", name: "Someone")
+    article = Article.new(id: 1, user: user, title: "New Things", body: "…are happening", created_at: DateTime.now, updated_at: DateTime.now)
+
+    schema = jbuilder_schema "api/v1/articles/_article",
```

Comment:
@newstler I can't figure out when this method is supposed to be used? Is it only for view contexts to call? If so, I think we could trim a lot of the internal code we're using to more-or-less rebuild Action View.'s rendering

Reply:
No, it's not only for view contexts, firstly it was for using in RSwag, i.e. in RSpec. Also that could be used in comparing API requests. Many other use cases probably. So basically in any place of our app we should be able to get the schema.

Reply:
@newstler cool, I'll try to see if there's a simpler to get at those things.